students/saleta_sebastian/lesson_07_string_datetime/to_swap_the_two_words.py

```diff
@@ -0,0 +1,17 @@
+def swap_two_words(user_word):
+    white_space_index = user_word.find(' ')
+    print(white_space_index)
+    if white_space_index != -1:
```

Comment:
According to the task description, it should not use loops and if. I know that this is for input validate but why you don't check if someone didn't input three or more words? In such cases, your script is not working. You can consider something like this:
```
    d = user_word.split()
    return ' '.join(d[::-1])
```

This will be also working when you will have more than two words.